tests/test_travis.yml

```diff
@@ -1,8 +1,7 @@
 language: python
 
 python:
-  - "3.5"
```

Comment:
Is this change required, as it is being used for testing the Travis modernizer script?

Reply:
I did it